tilt.py

```diff
@@ -73,8 +73,7 @@ def SetTiltRadioButtons(self):
         self.tiltbutton = []
         
         # Pull out the tilt indices and elevations for display
-        tilt_indices = self.Vradar.value.sweep_start_ray_index['data'][:]
-        elevs = self.Vradar.value.elevation['data'][tilt_indices]
+        elevs = self.Vradar.value.fixed_angle['data'][:]
```

Comment:
elevation is the real elevation for every ray, fixed_angle is the nominal target elevation